staticfuzz.py

```diff
@@ -282,6 +282,14 @@ def callback():
 
         return SlashCommandResponse(False, redirect)
 
+class SlashDrew(SlashCommand):
+    """Lillian if I messed this up It's because I  did it at 4 to try to make you smile and class inheritance is tricky~<3
+    """
+    NAME='drew'
+    @staticmethod
+    def(self);
+           message="Drew loves staticfuzz with all her heart~<3 uwu"
```

Comment:
This line will fail because of indentation error, same with line below. Use four spaces to denote a tab. Each scope adds an indent.